.datalad/crawl/crawl.cfg

```diff
@@ -0,0 +1,4 @@
+[crawl:pipeline]
+template = simple_with_stanford_lib
+_doc_id = vq183gc7199
```

Comment:
oh no -- we do not want this file in our code base -- I guess you ran `crawl-init` by mistake in this repository.  Please remove